Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.cpp

```diff
@@ -281,8 +281,23 @@ ARM64Reg Arm64FPRCache::R(u32 preg, bool only_lower)
 	switch (reg.GetType())
 	{
 	case REG_REG: // already in a reg
+		return reg.GetReg();
+	break;
 	case REG_LOWER_PAIR:
+	{
+		if (!only_lower)
```

Comment:
May you split this fix into another PR so we can merge it quickly? Feel free to push this lines directly, they LGTM.